lute/io/models/sfx_index.py

```diff
@@ -403,3 +403,19 @@ def validate_out_file(cls, out_file: str, values: Dict[str, Any]) -> str:
             fname: str = f"{expmt}_r{run:04d}.stream"
             return f"{work_dir}/{fname}"
         return out_file
+
+
+class MergeStreamFilesParameters(TaskParameters):
+
+    in_file: Optional[str] = Field(
+        "",
+        description="Root of directory tree storing stream files to merge.",
+        flag_type="-",
```

Comment:
Do you have a plan to use the `flag_type` and `rename_param` attributes in the `Task`? Maybe this is left-over from a draft as a third-party `Task`?

Reply:
It is, I will remove it!